File: src/main/java/Instruction.java
public class Instruction {
	public final byte id;
	public final String ainfo0;
	public final String ainfo1;
	
	public Instruction(byte id, String ainfo0, String ainfo1) {
		this.id = id;
		this.ainfo0 = ainfo0;
		this.ainfo1 = ainfo1;
	}
	
	@Override
	public String toString() {
		return "Instruction{" +
				"id=" + id +
				", ainfo0='" + ainfo0 + '\'' +
				", ainfo1='" + ainfo1 + '\'' +
				'}';
	}
}

File: native_vm/ConsoleApplication1/ConsoleApplication1/Instruction.cpp
#include "pch.h"
#include "Instruction.h"


Instruction::Instruction() {
}


Instruction::~Instruction() {
}

void Instruction::init() {
	if (isInt(ainfo0)) ainfo0I = stoi(ainfo0);
	if (isInt(ainfo1)) ainfo1I = stoi(ainfo1);
}
